ADR/ADR002-use-govuk-notify.md

```diff
@@ -4,7 +4,7 @@ Date: 2022-04-07
 
 ## Status
 
-Accepted
+Rejected
```

Comment:
Think this would be superseded rather than rejected? 
```suggestion
Superseded
```

Reply:
perf